suite-native/module-send/src/components/FeeOption.tsx

```diff
@@ -0,0 +1,86 @@
+import { useContext } from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { GeneralPrecomposedTransactionFinal } from '@suite-common/wallet-types';
+import { Text, HStack, VStack, Radio } from '@suite-native/atoms';
+import { CryptoToFiatAmountFormatter, CryptoAmountFormatter } from '@suite-native/formatters';
+import { FormContext } from '@suite-native/forms';
+import { TxKeyPath, Translation } from '@suite-native/intl';
+
+import { SendFeesFormValues } from '../sendFeesFormSchema';
+import { NativeSupportedFeeLevel } from '../types';
+
+const feeLabelsMap = {
+    economy: {
+        label: 'moduleSend.fees.levels.low.label',
+        timeEstimate: 'moduleSend.fees.levels.low.timeEstimate',
+    },
+    normal: {
+        label: 'moduleSend.fees.levels.medium.label',
+        timeEstimate: 'moduleSend.fees.levels.medium.timeEstimate',
+    },
+    high: {
+        label: 'moduleSend.fees.levels.high.label',
+        timeEstimate: 'moduleSend.fees.levels.high.timeEstimate',
+    },
+} as const satisfies Record<NativeSupportedFeeLevel, { label: TxKeyPath; timeEstimate: TxKeyPath }>;
+
+export const FeeOption = ({
+    feeKey,
+    feeLevel,
+    networkSymbol,
+}: {
+    feeKey: SendFeesFormValues['feeLevel'];
+    feeLevel: GeneralPrecomposedTransactionFinal;
+    networkSymbol: NetworkSymbol;
+}) => {
+    const { watch, setValue } = useContext(FormContext);
+    const selectedLevel = watch('feeLevel');
+
+    const isChecked = selectedLevel === feeKey;
+
+    const { label, timeEstimate } = feeLabelsMap[feeKey];
```

Comment:
I am testing this flow and the app crashes  on btc testnet send attempt, on `custom` fee label which is not allowed for native but is inserted via `SendFeesScreen params`

Reply:
https://github.com/user-attachments/assets/a07ea84d-ed58-427d-8cae-0c2e97797e57